doc/01_overview/compliance.rst

```diff
@@ -47,16 +47,17 @@ In addition, the following instruction set extensions are available.
      - 2.0
      - always enabled
 
-Most content of the RISC-V privileged specification is optional.
-Ibex currently supports the following features according to the RISC-V Privileged Specification, version 1.11.
+   * - **Smepmp** - PMP Enhancements for memory access and execution prevention on Machine mode
+     - 1.0
+     - always enabled in configurations with PMP see :ref:`PMP Enhancements<pmp-enhancements>`
+
+Ibex currently supports the following features according to the RISC-V Privileged Specification, version 1.12.
```

Comment:
I don't remember exactly but does Ibex now fully follow version 1.12 of the priv spec?

Reply:
Yes we switched to it a while ago https://github.com/lowRISC/ibex/issues/1733